### PR TITLE
Add Gabriel Fouasnon to SSC list

### PR DESCRIPTION
**Pull request by gabalafou**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -390,11 +390,12 @@
     twitter_handle: JohanMabille
     gh_handle: JohanMabille
     subproject: Jupyter Kernels
-  - name: Isabela Presedo-Floyd
-    photo: Isabela_Presedo-Floyd.jpg
```
Comment (gabalafou) [inline about.html:394]: Note I did not delete the photo from the repo because it is used in other places

**Comment by Zsailer** [comment]
LGTM! Thanks, @gabalafou and welcome to the team!

Test failures appear unrelated.

**Review by Zsailer — APPROVED** [review]

**Review by krassowski — APPROVED** [review]

**Comment by Zsailer** [comment]
Apparently, I don't have merge rights on this repo 😅 

ping @jupyter/executive-council  for review/merge.

**Comment by krassowski** [comment]
I restarted the failing jobs, they appear unrelated but want to give them another go before merging.

**Pull request merged**